examples/angular/src/app/pages/services/page.service.ts

```diff
@@ -45,54 +39,38 @@ export class PageService {
      * @return {*}  {(Observable<{ page: DotCMSPageAsset | { error: PageError }; nav: DotcmsNavigationItem }>)}
      * @memberof PageService
      */
-    getPageAndNavigation(route: ActivatedRoute, config: any): Observable<PageAndNavResponse> {
-        if (!this.navObservable) {
-            this.navObservable = this.fetchNavigation(route);
-        }
-
-        return forkJoin({
-            nav: this.navObservable,
-            pageAsset: this.fetchPage(route, config)
-        }).pipe(
-            map(({ nav, pageAsset }) => {
-                const { page, error } = pageAsset;
-
-                return { nav, page, error };
+    getPageAsset<TPage extends DotCMSPageAsset, TContent>(
+        route: ActivatedRoute,
+        extraQueries?: DotCMSPageRequestParams['graphql']
+    ): Observable<PageWithNavigation<TPage, TContent>> {
+        return this.fetchPage<TPage, TContent>(route, extraQueries).pipe(
```

Comment:
Just curious, why does `getPageAsset` call `fetchPage` with the same arguments?

Reply:
Not anymore, it was like that due to extra logic, but is not needed